specification/authorization/resource-manager/readme.md

```diff
@@ -115,6 +115,16 @@ input-file:
 - Microsoft.Authorization/preview/2018-01-01-preview/authorization-RoleBasedCalls.json
 ```
 
+### Tag: package-2018-09-01-preview
+
+These settings apply only when `--tag=package-2018-09-01-preview` is specified on the command line.
+
+``` yaml $(tag) == 'package-2018-09-01-preview'
+input-file:
+- Microsoft.Authorization/preview/2015-06-01/authorization-ClassicAdminCalls.json
+- Microsoft.Authorization/preview/2018-09-01-preview/authorization-RoleBasedCalls.json
+```
+
 ---
 # Code Generation

```

Comment:
add tag for GO like this
### Tag: package-2018-01-01-preview and go

These settings apply only when `--tag=package-2018-01-01-preview --go` is specified on he command line.
Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.

``` yaml $(tag) == 'package-2018-01-01-preview' && $(go)
output-folder: $(go-sdk-folder)/services/authorization/mgmt/2018-01-01-preview/authorization
```

Reply:
Is there a reason this is only done for go and not Java etc?